app/helpers/application_helper.rb

```diff
@@ -76,9 +76,11 @@ def admin_value(v)
 
   def admin_date(date, ago_only: false)
     ago_text = _('{{length_of_time}} ago', :length_of_time => time_ago_in_words(date))
-    return ago_text if ago_only
 
-    "#{date} (#{ago_text})"
+    ago_tag = content_tag(:span, ago_text, title: date)
```

Comment:
Instead of this, how about wrapping the output in a `time` tag like [`simple_date_html`](https://github.com/mysociety/alaveteli/blob/b1d999103889f6754d78d49e92cfad41e3302017/spec/helpers/date_time_helper_spec.rb#L33)? Haven't given this any thought as to whether that would work in this context so no big deal if it's a pain.

Reply:
Good call. Fixup pushed